code/modules/shuttle/supply.dm

```diff
@@ -1,5 +1,6 @@
 GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		/mob/living,
+		/mob/camera,
```

Comment:
can the ai now block shuttles by looking at them?

Reply:
Just cargo, and i doubt an ai would be that persistent.

Reply:
time to abuse

Reply:
Build a xenobio console there,  keep it stuck forever